packages/patternfly-4/react-core/src/components/LoginPage/LoginForm.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import { Form, FormGroup, ActionGroup, FormHelperText } from '../Form';
+import { TextInput } from '../TextInput';
+import { Button } from '../Button';
+import { Checkbox } from '../Checkbox';
+
+export interface LoginFormProps extends React.HTMLProps<HTMLFormElement> {
+  /** Additional classes added to the Login Main Body's Form */
+  className?: string;
+  /** Flag indicating the Helper Text is visible * */
+  showHelperText?: boolean;
+  /** Content displayed in the Helper Text component * */
+  helperText?: React.ReactNode;
+  /** Label for the Username Input Field */
+  usernameLabel?: string;
+  /** Value for the Username */
+  usernameValue?: string;
+  /** Function that handles the onChange event for the Username */
+  onChangeUsername?(value: string, event: React.FormEvent<HTMLInputElement>): void;
+  /** Flag indicating if the Username is valid */
+  isValidUsername?: boolean;
+  /** Label for the Password Input Field */
+  passwordLabel?: string;
+  /** Value for the Password */
+  passwordValue?: string;
+  /** Function that handles the onChange event for the Password */
+  onChangePassword?(value: string, event: React.FormEvent<HTMLInputElement>): void;
+  /** Flag indicating if the Password is valid */
+  isValidPassword?: boolean;
+  /** Label for the Log in Button Input */
+  loginButtonLabel?: string;
+  /** Flag indicating if the Login Button is disabled */
+  isLoginButtonDisabled?: boolean,
+  /** Function that is called when the Login button is clicked */
+  onLoginButtonClick?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
+  /** Label for the Remember Me Checkbox that indicates the user should be kept logged in.  If the label is not provided, the checkbox will not show. */
+  rememberMeLabel?: string;
+  /** Flag indicating if the remember me Checkbox is checked. */
+  isRememberMeChecked?: boolean;
+  /** Function that handles the onChange event for the Remember Me Checkbox */
+  onChangeRememberMe?: (checked: boolean, event: React.FormEvent<HTMLInputElement>) => void;
+  /** Aria Label for the Remember me checkbox, use this to override using the rememberMeLabel */
+  rememberMeAriaLabel?: string;
```

Comment:
We should decide across all components whether or not to keep the `console.error()` for things that used to be like
```js
props => {
    if (props.rememberMeLabel && !props.rememberMeAriaLabel) {
      return new Error('rememberMeAriaLabel is required with the Remember me checkbox');
    }
    return null;
  },```

Reply:
sure, but it this case it makes more sense to just the rememberMeLabel by default if no aria label is given. I would assume most apps would want them to be the same.